app/src/main/java/org/fossasia/openevent/app/core/event/about/AboutEventActivity.java

```diff
@@ -37,7 +37,10 @@ protected void onCreate(Bundle savedInstanceState) {
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case android.R.id.home:
-                finish();
+                if (getSupportFragmentManager().getBackStackEntryCount() > 0)
+                    getSupportFragmentManager().popBackStack();
+                else
+                    finish();
```

Comment:
Clicking on hardware back button should work as well

Reply:
It does. Since the fragment is added to backstack, clicking phone's back button works as well by reversing the transition.

Reply:
OK, use a shared variable for FragmentManager

Reply:
Umm, it's used only once. Why use a shared variable?